apis/v1beta1/validation/gateway.go

```diff
@@ -91,3 +92,18 @@ func validateListenerHostname(listeners []gatewayv1b1.Listener, path *field.Path
 	}
 	return errs
 }
+
+// validateTLSCertificateRefs validates the certificateRefs
+// must be set when tls config is set and TLSModeType is
+// terminate
+func validateTLSCertificateRefs(listeners []gatewayv1b1.Listener, path *field.Path) field.ErrorList {
+	var errs field.ErrorList
+	for i, h := range listeners {
```

Comment:
why `h` as the name for the current listener?

Reply:
Update it to `c`.